libs/markets/src/lib/market-utils.ts

```diff
@@ -51,15 +51,17 @@ export const getQuoteName = (market: Partial<Market>) => {
 
 export const sumFeesFactors = (fees: Market['fees']['factors']) => {
   return fees
-    ? new BigNumber(fees.makerFee)
-        .plus(fees.liquidityFee)
-        .plus(fees.infrastructureFee)
+    ? Number(fees.makerFee) +
```

Comment:
why this change was needed?

Reply:
This would be better left as BigNumber to avoid a floating point error..

Reply:
I've updated it to use BigNum